app/views/pages/moving-guide.html.erb

```diff
@@ -3,66 +3,66 @@
+    <div class="usa-media_block location-section">
+      <%= image_tag 'conus.svg', alt: 'Continental U S', class: 'usa-media_block-img' %>
+      <div class="usa-media_block-body">
+        <h3><%= abbr_tag('conus') %> (48 Contiguous States)</h3>
+        <p><%= abbr_tag('conus') %> are moves that start and end within the 48 contiguous states of the US. Most moves in the US Military are <%= abbr_tag('conus') %> moves.</p>
+        <p><%= link_to raw("What to expect for a #{abbr_tag('conus')} move?"), page_path('moving-guide/conus') %></p>
+      </div>
+    </div>
+
+    <div class="usa-media_block location-section">
+      <%= image_tag 'oconus.svg', alt: 'Outside Continental U S', class: 'usa-media_block-img' %>
+      <div class="usa-media_block-body">
+        <h3><%= abbr_tag('oconus') %> (International + Hawaii &amp; Alaska)</h3>
+        <p>Moves to or from Alaska and Hawaii along with international stations are considered <%= abbr_tag('oconus') %> moves. These moves are more complicated and each location may have additional limitations on what you are allowed to transport.</p>
+        <p><%= link_to raw("What to expect for a #{abbr_tag('oconus')} Move?"), page_path('moving-guide/oconus') %></p>
+      </div>
+    </div>
+
+    <h2 id="move-types">Know Your Options. (Move Types)</h2>
+
+    <p>Once you have figured out the type of move (<%= abbr_tag('pcs') %> or <%= abbr_tag('tdy') %>) and your move location (<%= abbr_tag('conus') %> or <%= abbr_tag('oconus') %>), you will have some choices to make regarding how your belongings are shipped to your new destination. Remember, you can split up your total allowable weight into multiple shipments. For example, most service members doing a <%= abbr_tag('pcs') %> <%= abbr_tag('conus') %> move will have both a <%= abbr_tag('ppm') %> (“do it yourself”) shipment of their prized, valuable possessions and an <%= abbr_tag('hhg') %> (“Household Goods") shipment using a government-furnished moving company for the bulk of their belongings.</p>
     <p>If you have any questions, your local personal property office is the place to go for help figuring out how to break up your weight into various shipments.</p>
 
     <h3>Move Types</h3>
 
-    <table class="usa-table-borderless">
-      <thead>
-        <tr>
-          <th scope="col">Move Type</th>
-          <th scope="col">Description</th>
-        </tr>
-      </thead>
+    <table>
       <tbody>
         <tr>
-          <th scope="row">Personally-Procured Move (PPM)</th>
+          <th scope="row"><%= image_tag 'ppm.svg', alt: 'P P M', role: 'presentation' %><strong>Personally Procured Move</strong> (PPM)</th>
```

Comment:
Since these images are presentation (as conveyed with `role="presentation"`), it's safe to set an empty `alt=""` on these elements.

In fact… it may be _better_ to set the empty `alt=""` for browsers and assistive technologies that don't pick up on the `role="presentation"` attribute.

So, something like:

```erb
<%= image+tag 'ppm.svg', alt: '', role: 'presentation' %>
```